.github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml

```diff
@@ -0,0 +1,27 @@
+name: AVM - CI Environment Issue 🚀
+description: Have an issue, question or suggestion for the CI Environment? Let us know!
+title: "[AVM CI Environment Issue]: "
+labels: ["Type: AVM :a: :v: :m:", "Needs: Triage :mag:"]
```

Comment:
```suggestion
labels: ["Type: AVM :a: :v: :m:", "Needs: Triage :mag:", "Needs: Core Team :genie:"]
```

Reply:
CI label to be added too as soon as created

Reply:
Added that too. 👍